efsity/src/main/kotlin/org/smartregister/external/FhirCarePlanGeneratorLite.kt

```diff
@@ -274,7 +274,7 @@ class FhirCarePlanGeneratorLite {
   private val fhirPathEngine: FHIRPathEngine =
     with(FhirContext.forCached(FhirVersionEnum.R4)) {
       FHIRPathEngine(HapiWorkerContext(this, this.validationSupport)).apply {
-        hostServices = FhirPathEngineHostServices
+        hostServices = FHIRPathEngineHostServices
```

Comment:
the standard in camelcase for acronyms is to only capitalize the first character, please revert